docker-compose.yml

```diff
@@ -9,27 +9,27 @@ services:
       - ./pg_data:/var/lib/postgresql
   app:
     build: .
-    entrypoint: /opt/cloudigrade/entrypoint.sh
+    entrypoint: /opt/entrypoint.sh
     environment:
       - DJANGO_SETTINGS_MODULE=config.settings.docker
       - AWS_ACCESS_KEY_ID
       - AWS_SECRET_ACCESS_KEY
       - AWS_DEFAULT_REGION
     volumes:
       - ./cloudigrade:/opt/cloudigrade:Z
-      - ./docker/entrypoint.sh:/opt/cloudigrade/entrypoint.sh:Z
+      - ./docker/entrypoint.sh:/opt/entrypoint.sh:Z
       - /tmp/cloudigrade/socket:/var/run/cloudigrade:Z
-      - static:/opt/cloudigrade/static
+      - /tmp/cloudigrade/static:/srv/cloudigrade/static:Z
     depends_on:
       - db
   web:
     image: nginx:alpine
     volumes:
       - ./docker/nginx.conf:/etc/nginx/nginx.conf:Z
       - /tmp/cloudigrade/socket:/var/run/cloudigrade:Z
-      - static:/opt/cloudigrade/static
+      - /tmp/cloudigrade/static:/srv/cloudigrade/static:Z
```

Comment:
Does something need to `mkdir` this first?

Reply:
Compose handles it.